src/compoz.ts

```diff
@@ -267,6 +271,40 @@ export class Compoz {
 		this.elBSendImg.src = svgSendDisable
 	}
 
+	getLastValue(arr: string[]) {
+		return arr[arr.length - 1]
+	}
+
+	resetHistoryChanges() {
+		this.historyChanges = []
+		this.historyChanges.push("")
+		this.redoChanges = []
+		this.oldChange = ""
+	}
+
+	undoContent() {
+		if (this.historyChanges.length === 1) {
```

Comment:
Why 1 not 0?

Reply:
Because in the first array of historyChanges, I have initialized with (""). I make this because I want to handle when user undoes until the first historyChanges, they still can redo and the oldChanges never have an undefined value.